Assets/MixedRealityToolkit/_Core/Interfaces/SpatialAwareness/IMixedRealitySpatialAwarenessSystem.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.SpatialAwareness;
+//using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
+//using Microsoft.MixedReality.Toolkit.Internal.Interfaces.Events;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Interfaces.SpatialAwarenessSystem
+{
+    public interface IMixedRealitySpatialAwarenessSystem // : IMixedRealityEventSystem, IMixedRealityEventSource
```

Comment:
Depending on how we implement this, the event source could be whatever we use to wrap the spatial understanding dll with, and we will raise events from it to the spatial awareness system.

That way we can have multiple sources of spatial data, but a single spatial awareness system.